### PR TITLE
Change MerkleDropMinter to use back individual mint counts per account per drop

### PR DESCRIPTION
**Pull request by Vectorized**

### CODE REVIEW

File: contracts/modules/MerkleDropMinter.sol [hunk 1]
```diff
@@ -26,6 +26,14 @@ contract MerkleDropMinter is IMerkleDropMinter, BaseMinter {
      */
     mapping(address => mapping(uint128 => EditionMintData)) internal _editionMintData;
 
+    /**
+     * @dev The number of mints for each account.
+     *      Maps `edition` => `mintId` => `address` => value.
+     *      We will simply store a uint256 for every account, to keep the Merkle tree
+     *      simple, so that it is compatible with 3rd party allowlist services like Lanyard.
+     */
+    mapping(address => mapping(uint128 => mapping(address => uint256))) internal _mintCounts;
```
Comment (gigamesh) [inline contracts/modules/MerkleDropMinter.sol:35]: Don't you need to add the logic to update this during `mint()` or are you planning to do that in a follow up?

**Review by gigamesh — APPROVED** [review]
nice!

File: contracts/modules/MerkleDropMinter.sol (hunk 1 above)
Comment (vigneshka) [inline contracts/modules/MerkleDropMinter.sol:35]: how can client know how many user is eligible to mint?
should this be public?

**Review by vigneshka — CHANGES_REQUESTED** [review]

File: contracts/modules/MerkleDropMinter.sol (hunk 1 above)
Comment (Vectorized) [reply contracts/modules/MerkleDropMinter.sol:35]: Good catch.

**Review by gigamesh — APPROVED** [review]

**Pull request merged**